modules/serverless-kafka-sasl-source.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sasl-source_{context}"]
+= Configuring SASL authentication for Kafka sources
+
+_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed.
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {product-title}.
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` CR are installed on your {product-title} cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have a username and password for a Kafka cluster.
+* You have chosen the SASL mechanism to use, for example `PLAIN`, `SCRAM-SHA-256`, or `SCRAM-SHA-512`.
+* If TLS is enabled, you also need the `ca.crt` certificate file for the Kafka cluster.
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+[source,terminal]
+----
+$ oc create secret -n <namespace> generic <kafka_auth_secret> \
+  --from-file=ca.crt=caroot.pem \
+  --from-literal=password="SecretPassword" \
+  --from-literal=saslType="SCRAM-SHA-512" \
+  --from-literal=user="my-sasl-user"
+----
+** Use the key names `ca.crt`, `password`, and `sasl.mechanism`. Do not change them.
```

Comment:
This note isn't technically correct for KafkaSource since we can reference different keys in the secret as described at line 97 [1]

```yaml
      user:
        secretKeyRef:
          name: <secret-name>
          key: user
```